code/player/TTTPlayer.AFK.cs

```diff
@@ -0,0 +1,41 @@
+using System.Linq;
+
+using Sandbox;
+
+using TTTReborn.Settings;
+
+namespace TTTReborn.Player
+{
+    public partial class TTTPlayer
+    {
+        private static int SecondsTillKick => ServerSettings.Instance.AFK.MinutesTillKick * 60;
+        private float TimeToKick => _timeSinceLastAction + SecondsTillKick;
+
+        private float _timeSinceLastAction = 60.0f;
+        private Rotation? LastKnownRotation;
+
+        private void TickAFKSystem()
+        {
+            bool pressedAnyKeyPressed = Buttons.Any(button => Input.Pressed(button) || Input.Down(button));
+
+            if (pressedAnyKeyPressed || (LastKnownRotation.HasValue && LastKnownRotation != Rotation))
+            {
+                _timeSinceLastAction = Time.Now;
+                LastKnownRotation = Rotation;
+            }
+
+
+            if (TimeToKick < Time.Now)
```

Comment:
I think it would be better to use `TimeSince` variable.

```suggestion
        private static int SecondsTillKick => ServerSettings.Instance.AFK.MinutesTillKick * 60;
        
        private TimeSince _timeSinceLastAction = 0f;
        private Rotation? LastKnownRotation;

        private void TickAFKSystem()
        {
            bool pressedAnyKeyPressed = Buttons.Any(button => Input.Pressed(button) || Input.Down(button));

            if (pressedAnyKeyPressed || (LastKnownRotation.HasValue && LastKnownRotation != Rotation))
            {
                _timeSinceLastAction = 0;
                LastKnownRotation = Rotation;
            }
            if (SecondsTillKick < _timeSinceLastAction)
```

Reply:
I have switched to TimeSince. Thank you.